php_bson.h

```diff
@@ -60,11 +69,7 @@ typedef struct {
 	bool                     is_visiting_array;
 } php_phongo_bson_state;
 
-#if PHP_VERSION_ID >= 70000
-#define PHONGO_BSON_STATE_INITIALIZER  { {{ 0 }}, { PHONGO_TYPEMAP_NONE, NULL, PHONGO_TYPEMAP_NONE, NULL, PHONGO_TYPEMAP_NONE, NULL}, NULL, 0 }
-#else
-#define PHONGO_BSON_STATE_INITIALIZER  { NULL, { PHONGO_TYPEMAP_NONE, NULL, PHONGO_TYPEMAP_NONE, NULL, PHONGO_TYPEMAP_NONE, NULL}, NULL, 0 }
-#endif
+#define PHONGO_BSON_STATE_INITIALIZER  { 0 }
```

Comment:
Note that we're already using `{0}` for initializing potentially nested structs (e.g. `smart_str` in PHP 7: https://github.com/mongodb/mongo-php-driver/blob/608ce77839e1760f9a7272aa61d1158d6bcd741c/src/BSON/ObjectID.c#L199) and we've received no complaints about build issues.

http://stackoverflow.com/a/1761873/162228 suggests that this is kosher.